crowbar_framework/spec/models/service_object_spec.rb

```diff
@@ -62,7 +62,7 @@
 
     it "validates the proposal" do
       prop = proposal
-      CrowbarValidator.any_instance.expects(:validate).with(prop.raw_data).once.returns([])
+      expect_any_instance_of(CrowbarValidator).to receive(:validate).with(prop.raw_data).and_return([])
```

Comment:
Metrics/LineLength: Line is too long. [103/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)